fix: Season, Episode, and Previously Shown

jesmannstl opened the pull request:
<!-- Please use Conventional Commits to label your title -->
<!-- https://www.conventionalcommits.org/en/v1.0.0/ -->
<!-- Example: feat: allow provided config object to extend other configs  -->

### Description

<!-- Fixes #58  -->
<!-- Please also include relevant motivation and context. -->

### Testing

<!-- Please describe the tests that you ran to verify your changes. -->
<!-- Provide instructions so we can reproduce. -->
<!-- Please also list any relevant details for your test configuration -->

### New dependencies

<!-- List any dependencies that are required for this change. -->
<!-- Otherwise, delete section. -->

jesmannstl committed:
Update tvlistings.ts

Corrected for Season/Episode Info

jesmannstl committed:
Update xmltv.ts

Correct Season/Episode Display and Previously Shown

jesmannstl committed:
Update tvlistings.ts

Corrected Season/Episode Numbering

jesmannstl committed:
Update README.md

jesmannstl committed:
Update README.md

jesmannstl committed:
Update tvlistings.ts

Removed w=55 from channel image url

jesmannstl committed:
Add files via upload

Added <category> tag if available and optional add Series <category> tag to non-movies without a category

jesmannstl committed:
Update README.md

Add --includeSeriesGenre info

jesmannstl commented:
This now includes `<category>` when available and optional `--includeSeriesGenre` to add `<category>Series</category>` on Series without a category.

jef commented:
> This now includes <category> when available and optional --includeSeriesGenre to add <category>Series</category> on Series without a category.

Do you think that should be default behavior without the flag?

jesmannstl commented:
It was optional on the perl script so I was working with the same logic.

jesmannstl committed:
Made add Series to non-movie by default

jesmannstl committed:
Removed --includeSeriesGenre option as built in

jesmannstl commented:
> > This now includes  when available and optional --includeSeriesGenre to add Series on Series without a category.
> 
> Do you think that should be default behavior without the flag?

Updated to be default behavior.  I was unable to update the "test" files successfully to automatically build

jesmannstl committed:
Update tvlistings.ts

roblatour committed:
updates

jesmannstl reviewed (commented):
Updated aid, Season, Episode numbering, New, Live, Premiere, Previously shown, Add Series, News, Sports, Family etc  tags to non-movie, other additions

jesmannstl committed:
Update README.md

Include "Series" category on series now automatic

roblatour committed:
updates

roblatour committed:
Update config.ts

roblatour committed:
Merge upstream changes and resolve README conflicts - Remove --includeSeriesGenre option (now built-in) - Keep Docker improvements and documentation

jesmannstl committed:
Merge pull request #3 from roblatour/main

Pulling small changes

jesmannstl committed:
Update <date> add. ep. naming --addAsterisk option

jesmannstl committed:
Update <date> add. ep. naming --addAsterisk option

jesmannstl committed:
Update README.md

jesmannstl committed:
Update README.md --appendAsterisk

jesmannstl committed:
Minor correction

jesmannstl committed:
Added SYYYYEMMDD to Series without any ex (News)

jesmannstl committed:
Update ci.yaml

jesmannstl committed:
Delete .github/workflows/ci.yaml

jesmannstl committed:
Create ci.yml

jesmannstl committed:
Create docker-image.yml

jesmannstl committed:
Update README.md

Updated docker to use updated fork

jesmannstl committed:
Update nightly-release.yaml

jesmannstl committed:
Update nightly-release.yaml

jesmannstl committed:
Update README.md

jesmannstl committed:
Update Season Episode if not Movie/Sports

jesmannstl committed:
Updated Changelog with all updates since 7/22

jesmannstl committed:
Update xmltv.ts

jesmannstl committed:
Added changes on this fork

jesmannstl committed:
Update README.md with recent updates

jesmannstl committed:
Update README.md with recent updates

jesmannstl committed:
Added many countries and mediaportal option

jesmannstl committed:
Updated options and country

jesmannstl committed:
Update Valid Country Codes.md

jesmannstl committed:
Update README.md

jesmannstl committed:
Removed icon width for better viewing

jesmannstl committed:
Merge branch 'main' of https://github.com/jesmannstl/zap2xml

jesmannstl committed:
Update README.md

jesmannstl committed:
Update README.md

jesmannstl committed:
Updated date tag added length

jesmannstl committed:
Update README.md

jesmannstl committed:
Update FUNDING.yml

jesmannstl committed:
Postal Code not required except OTA

jesmannstl committed:
Reordered Program fields to match original Perl output order

jesmannstl committed:
Fixed Sorting by channel/time

jesmannstl committed:
Fixed "dd_progid" tag

github-actions[bot] committed:
chore(main): release 2.2.1 (#1)

Co-authored-by: github-actions[bot] <41898282+github-actions[bot]@users.noreply.github.com>

jesmannstl committed:
Update README.md with Lineup links

jesmannstl committed:
Update README.md With lineup links

jesmannstl committed:
Update tvlistings order to fix bad request

jesmannstl committed:
Added Display Name with Channel Number and Call Sign

jesmannstl committed:
Added --nextpvr option

jesmannstl committed:
Update CHANGELOG.md

jesmannstl committed:
Changed default sort add sort options

jesmannstl committed:
Merge branch 'main' of https://github.com/jesmannstl/zap2xml

jesmannstl committed:
Update FUNDING.yml

Fillimerica committed:
Enhanced buildProgramsXml to check neighboring entries for exact object level duplicate. If a duplicate then the trailing object is not added into the xml stream. (#17)

Needed to add an option to the buildoptions to support the import of the assert module from NodeJS.

Co-authored-by: Thomas Filliman <fillimerica@yahoo.com>

jesmannstl committed:
image replacement

jesmannstl committed:
image server
